spaceship/lib/spaceship/tunes/iap.rb

```diff
@@ -52,6 +52,16 @@ def families
       # @param family_id (String) Only used on RECURRING purchases, assigns the In-App-Purchase to a specific familie
       # @param subscription_free_trial (String) Free Trial duration (1w,1m,3m....)
       # @param subscription_duration (String) 1w,1m.....
+      # @param subscription_price_target (Hash) Only used on RECURRING purchases, used to set the
+      # price of all the countries to be roughly the same as the price calculated from the price
+      # tier and currency given as input.
+      # @example:
+      #  [
+      #    {
+      #      currency: "EUR",
+      #      tier: 2
+      #    }
+      #  ]
```

Comment:
Oh, this is where I got the array from ^ Should this example really just be a hash?

Reply:
Oh right! Fixed, my bad 👍